### PR TITLE
CHANGELOG: release 4.26.0 and add notes for the next release

### PR DESCRIPTION
**Pull request by benma**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,13 @@
 # Changelog
 
 ## [Unreleased]
+- Buy ERC20 tokens using Moonpay
+- Remove CryptoCompare; use Coingecko for latest conversion rates. Fixes rate limiting issues, especially for VPN/Tor users.
+- Bundle BitBox02 v9.5.0 firmware
+- Run BitBoxApp installer as admin by default on Windows
+- Close a running BitBoxApp instance on Windows when installing an update to ensure success
+
```
Comment (x1ddos) [inline CHANGELOG.md:9]: Consider adding another item: More descriptive error when TOR connection fails.

**Review by x1ddos — APPROVED** [review]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (benma) [reply CHANGELOG.md:9]: Saw too late: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1219

You can also "request changes" so I can't merge :)

**Comment by x1ddos** [comment]
> Saw too late: 
https://github.com/digitalbitbox/bitbox-wallet-app/pull/1219
 > You can also "request changes" so I can't merge :)

I have a simpler and more effective solution for you: just read the 
comments before merging :)

**Pull request merged**